crescendo_2024/src/main/java/frc/robot/subsystems/ShootSubsystem.java

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkLowLevel;
+
+public class ShootSubsystem extends SubsystemBase {
+
+  private static CANSparkMax topMotor = new CANSparkMax(Constants.ShooterConstants.TopMotorCANID, CANSparkLowLevel.MotorType.kBrushless);
+  private static CANSparkMax bottomMotor = new CANSparkMax(Constants.ShooterConstants.BottomMotorCANID, CANSparkLowLevel.MotorType.kBrushless);
+
+
+  public ShootSubsystem() {
+
+    topMotor.restoreFactoryDefaults();
```

Comment:
if youre going to restore factory defaults you might want to add in all your settings but that shouldn't matter for right now